test/gurka/test_ferry_connections.cc

```diff
@@ -261,9 +259,7 @@ TEST(Standalone, ReclassifyFerryUntagDestOnly) {
         {"motorcycle", "yes"},
         {"route", "ferry"}}},
       {"EF", not_destonly},
-      {"FX", destonly}, // should stay destonly and low-class bcs connecting edge isn't and this will
```

Comment:
before there was another edge necessary to not have 2 edges with the same start/end node. luckily https://github.com/valhalla/valhalla/pull/4361 fixes that